src/scripts/main.js

```diff
@@ -1,3 +1,350 @@
 'use strict';
 
-// write code here
+// sorting by clicking on the title (in two directions)
+
+const tBody = document.querySelector('tbody');
+
+const data = [...tBody.children];
+
+const titles = document.querySelectorAll('th');
+
+function sortFnAsc(cellIndex) {
+  data.sort((a, b) => {
+    const contentA = a.cells[cellIndex].innerHTML;
+    const contentB = b.cells[cellIndex].innerHTML;
+
+    switch (cellIndex) {
+      case 3:
+        return contentA - contentB;
+      case 4:
+        return getNumber(contentA) - getNumber(contentB);
+      default:
+        return contentA.localeCompare(contentB);
+    }
+  });
+};
+
+function sortFnDesc(cellIndex) {
+  data.sort((a, b) => {
+    const contentA = a.cells[cellIndex].innerHTML;
+    const contentB = b.cells[cellIndex].innerHTML;
+
+    switch (cellIndex) {
+      case 3:
+        return contentB - contentA;
+      case 4:
+        return getNumber(contentB) - getNumber(contentA);
+      default:
+        return contentB.localeCompare(contentA);
+    }
+  });
```

Comment:
You do the same things in both functions 🤔  Can you combine it? For example add a sort type to a function arguments 
```
function sortRows(cellIndex, sortType) {
  const contentA = sortType === 'asc' : ... ? ... ;